.registryrc

```diff
@@ -1 +1 @@
-d55a6e9b3502d319eb9c395c644da8587f2b0f2d
+a3180f7a93f98079859333f837559c4abfad6430
```

Comment:
i assume this will be updated after your PR is merged to main?

Reply:
yup